library/ssl_tls12_server.c

```diff
@@ -1510,10 +1510,9 @@ static int ssl_parse_client_hello(mbedtls_ssl_context *ssl)
             MBEDTLS_TLS_SIG_NONE
         };
 
-#if defined(static_assert)
```

Comment:
NOTE: since there was no `#include <assert.h>` at the top of this file, `defined(static_assert)` would always be false, so this check was never going to happen - just showing that we should have an "always-present" static assert like this